src/components/client/signinButton.tsx

```diff
@@ -2,11 +2,8 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-'use client'
-
-import { signIn, useSession } from "next-auth/react";
+"use client";
```

Comment:
TIL: https://nextjs.org/docs/getting-started/react-essentials#the-use-client-directive